Formula/leafnode.rb

```diff
@@ -14,6 +14,89 @@ def install
                           "--with-user=#{ENV["USER"]}", "--with-group=admin",
                           "--sysconfdir=#{etc}/leafnode", "--with-spooldir=#{var}/spool/news/leafnode"
     system "make", "install"
+    (prefix/"homebrew.mxcl.fetchnews.plist").write fetchnews_plist
+    (prefix/"homebrew.mxcl.texpire.plist").write texpire_plist
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
This caveat does show up, in addition to `Homebrew` generated plist caveat for the main executable.

```
==> Caveats
For starting fetchnews and texpire, create links,
  ln -s /usr/local/opt/leafnode/homebrew.mxcl.{fetchnews,texpire}.plist ~/Library/LaunchAgents
And to start the services,
  launchctl load -w ~/Library/LaunchAgent/homebrew.mxcl.{fetchnews,texpire}.plist

To have launchd start leafnode now and restart at login:
  brew services start leafnode
Or, if you don't want/need a background service you can just run:
  leafnode
```